### PR TITLE
feat: add `KubectlLayer`

### PR DESCRIPTION
**Pull request by rix0rrr**
This can be plugged into the `kubectlLayer` argument of EKS's `Cluster` construct.

Fixes #

### CODE REVIEW

File: src/kubectl-asset.ts [hunk 1]
```diff
@@ -1,26 +1,16 @@
-import * as crypto from 'crypto';
-import * as fs from 'fs';
-import * as path from 'path';
 import * as s3_assets from 'aws-cdk-lib/aws-s3-assets';
 import { Construct } from 'constructs';
+import { assetHash, ASSET_FILE } from './_asset';
```
Comment (madeline-k) [inline src/kubectl-asset.ts]: Why not just remove this file?

**Review by madeline-k — DISMISSED** [review]

File: src/kubectl-asset.ts (hunk 1 above)
Comment (rix0rrr) [reply src/kubectl-asset.ts]: We could. It's breaking, but we don't care?

Comment (madeline-k) [reply src/kubectl-asset.ts]: Well, we are also gonna change the name of the package! So it's not breaking.

**Review by madeline-k — COMMENTED** [review]
LGTM now, but I am also authoring now. So will try to get approval from someone else before merging.

**Review by kaizencc — APPROVED** [review]

**Pull request merged**